src/auth.ts

```diff
@@ -47,6 +48,9 @@ export const drive = google.drive({
   version: 'v3',
   auth: oauth2Client,
 }) as Drive;
+export const discovery = google.discovery({
```

Comment:
@grant not sure if this should be in the `auth` file, as you don't need to authenticate to use it. However, it made sense at the time to put it there.